lib/jsonapi/query_parser.ex

```diff
@@ -65,6 +65,8 @@ defmodule JSONAPI.QueryParser do
     |> parse_include(Map.get(query_params, "include", ""))
     |> parse_filter(Map.get(query_params, "filter", %{}))
     |> parse_sort(Map.get(query_params, "sort", ""))
+    |> Map.put(:limit, Map.get(query_params, "limit", ""))
```

Comment:
If we're going to add these, we should add all the query params JSON API supports for pagination:

+ limit
+ offset
+ size
+ page
+ cursor

These should probably return integers and not strings with `nil` being a default and not `""`, don't you think?

We can probably refactor this whole pipe when we do add the aforementioned params to be a big cleaner and avoid the nested function calls.

Reply:
Yeah, I'll buy that -- side note is there a native ecto syntax for `cursor`?

Reply:
Main question being, what values are we expecting that to come across with?